src/DistributedMetadata/TaskStatusService.cpp

```diff
@@ -558,13 +554,14 @@ bool TaskStatusService::createTaskTable()
     ContextPtr context = Context::createCopy(global_context);
     context->setCurrentQueryId("");
     context->setQueryParameter("_payload", query_payload);
+    context->setUser("system", context->getPasswordByUserName("system"), Poco::Net::SocketAddress("127.0.0.1", 0));
     context->setDistributedDDLOperation(true);
     CurrentThread::QueryScope query_scope{context};
 
     try
     {
-        auto stream = executeQuery(query, context, true, QueryProcessingStage::Enum::WithMergeableStateAfterAggregation, false);
-        stream.onFinish();
+        executeSelectQuery(
+            query, context, [](Block &&) {}, true, QueryProcessingStage::Enum::WithMergeableStateAfterAggregation);
```

Comment:
It seems to me  `WithMergeableStateAfterAggregation` is a wrong here. It shall be `Complete`